apps/dashboard/app/apps/manifest.rb

```diff
@@ -101,6 +101,13 @@ def url
     @manifest_options[:url] || ""
   end
 
+  # Return ssh_allow Boolean
+  #
+  # @return [Boolean]
+  def ssh_allow
+    @manifest_options[:ssh_allow].nil? ? true : @manifest_options[:ssh_allow]
+  end
```

Comment:
I think we should be grabbing this out of the form. Whether it's the best choice or not I think is a valid question, but right now we have crammed a lot of other extra stuff in the `form.yml`.

So my thinking is, let's continue that even if it's not the best decision - at least all that extra stuff will be in 1 place (the `form.yml`) should we decide to extract it all and put in somewhere else.